salt/loader.py

```diff
@@ -42,6 +42,12 @@
 except ImportError:
     HAS_PKG_RESOURCES = False
 
+try:
+    import salt.utils.jinja
```

Comment:
This might be a bit too much, but I want to play it safe: in case there's an issue with the Jinja utils module we should not break everything.